### ISSUE TITLE
ddvk-hacks doesn't re-start after update

### ISSUE THREAD

**Issue by Snowflake6** [matteodelabre/toltec#649]
After updating from 2.1.14.something to 2.1.15.latest, ran toltecctl reenable to get everything going again.  Unfortunately, ddvk-hacks was not applied.  It was necessary to remove remarkable-stylus, remove ddvk-hacks, then re-install ddvk-hacks, and re-ininstall remarkable-stylus, in order to get the xochitl binary updated.

Sorry, logs not kept.  This was the same situation after the last 2.1.14.lower to 2.1.14.higher update as well, for some reason I never made it back here to log an issue.

**Comment by matteodelabre** [comment]
Seems like expected (yet unwanted) behavior, as ddvk-hacks does not install anything in /opt? (#539)

**Comment by Eeems** [comment]
If that's all it is, there is an easy solution someone just needs to open a pr to apply that kernelctl already does.

**Comment by Snowflake6** [comment]
toltecctl reenable seems to go through and install a number of packages, one of which is ddvk-hacks... But ddvk-hacks appears to fail when run via toltecctl reenable.  Is that expected?

**Comment by matteodelabre** [comment]
Are there any logs when it fails?

**Comment by Snowflake6** [comment]
I didn't notice the failure until after the reboot so I don't know.  Can I pull those logs somehow after-the-fact?

Actually, looking at #539 again it's probably the same issue... I don't explicitly remember it trying to re-install ddvk-hacks when I did the reenable this time, but the behaviour was identical... I had to remove remarkable-stylus, remove ddvk-hacks, re-install ddvk-hacks, and re-install remarkable-stylus.
